packages/ember-glimmer/tests/integration/helpers/inline-if-test.js

```diff
@@ -0,0 +1,144 @@
+import { moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import EmberObject from 'ember-runtime/system/object';
+import jQuery from 'ember-views/system/jquery';
+
+import {
+  BASIC_TRUTHY_TESTS,
+  BASIC_FALSY_TESTS,
+  SharedHelperConditionalsTest
+} from '../../utils/shared-conditional-tests';
+
+moduleFor('Helpers test: inline {{if}}', class extends SharedHelperConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{if ${cond} ${truthy} ${falsy}}}`;
+  }
+
+  ['@htmlbars it can omit the falsy argument']() {
+    this.render(`{{if condition 'yes'}}`, { condition: true });
+
+    this.assertText('yes');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('yes');
+
+    this.runTask(() => set(this.context, 'condition', false));
+
+    this.assertText('');
+
+    this.runTask(() => set(this.context, 'condition', true));
+
+    this.assertText('yes');
+  }
+
+  ['@test it can use falsy param as binding']() {
+    this.render(`{{if condition truthy falsy}}`, { condition: false, truthy: 'yes', falsy: 'no' });
```

Comment:
I think this is the "start with `false`" version of the "static argument" test? In general, we try to group them in a single test (see the comment above) so that we don't have an explosion of test cases. But this is also related to the broader point of https://github.com/emberjs/ember.js/pull/12920/files#r53242487